lib/notifications/notification.ex

```diff
@@ -88,4 +91,26 @@ defmodule Notifications.Notification do
   defp log_creation(notification) do
     Logger.warn("Notification created new_notification=#{inspect(notification)}")
   end
+
+  defp link_notification_to_users(notification) do
+    now = NaiveDateTime.utc_now() |> NaiveDateTime.truncate(:second)
+
+    user_ids = User.user_ids_for_route_ids(notification.route_ids)
+
+    notification_user_maps =
+      Enum.map(user_ids, fn user_id ->
+        %{
+          notification_id: notification.id,
+          user_id: user_id,
+          state: :unread,
+          inserted_at: now,
+          updated_at: now
```

Comment:
Very minor thing, but is there some way to use the current time from the database? If so that might save us a call to `utc_now` and so on.

Reply:
There are current-time functions, but then we'd have to re-write this insert as a mess of SQL fragments and bindings. This way is simpler and easier to follow than that would be, IMHO.

Reply:
Fair - for some reason I thought there was an easier way of accomplishing that besides fragments but if so I'm not able to dig it up currently.